src/hssm/config.py

```diff
@@ -86,19 +88,21 @@ def from_defaults(
                     loglik_config = default_config["likelihoods"][loglik_kind]
                     return Config(
                         model_name,
-                        loglik_kind,
+                        loglik_kind=loglik_kind,
+                        response=default_config["response"],
                         list_params=default_config["list_params"],
                         description=default_config["description"],
                         **loglik_config,
                     )
                 return Config(
                     model_name,
-                    loglik_kind,
+                    loglik_kind=loglik_kind,
+                    response=default_config["response"],
                     list_params=default_config["list_params"],
                     description=default_config["description"],
                 )
 
-            return Config(model_name, loglik_kind)
+            return Config(model_name, loglik_kind, response=["rt", "response"])
```

Comment:
The logic here is that when "model_name" supplied is not in any of the supported models (meaning the user is using a custom model), we by default hard-code `response` to be `["rt", "response"]`. This will get overridden by the `response` field in the `model_config` that they supply. This way, we don't require them to define `response` fields for compatibility with existing code base